### PR TITLE
Isolated Stamper v1

### PR DESCRIPTION
**Pull request by rstai1130**
Isolated Stamper v1

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -332,7 +342,7 @@ task source(dependsOn: src)
 task compile(dependsOn: src) 
 
 // Forcing srcBeautify on src step
-src.dependsOn srcBeautify
+//src.dependsOn srcBeautify
```
Comment (PicoCreator) [inline build.gradle]: Add this back in, for the latest version of master - there is already code in place (for this branch) to skip the step for windows.

File: build.gradle [hunk 2]
```diff
@@ -76,9 +76,19 @@ dependencies {
...
 	//-------------------------------------------------------------------------------------------------------------------
 
+	// iText 
+	api files("./lib/iText.jar")
+
+	//batik SVG
+	api fileTree(dir: './lib/batik/', include: ['*.jar']) 
+
+	// https://mvnrepository.com/artifact/org.w3c.dom/svg
+	implementation 'org.kie.modules:org-apache-batik:6.1.0.Beta1'
```
Comment (PicoCreator) [inline build.gradle:91]: If your adding bartik library here, why are u commiting the jars as well? 

Please remove the uneeded bartik jars if this is used instead.

File: README.md [hunk 3]
```diff
@@ -5,4 +5,5 @@ Collection of PDF manipulation utilities
 # Developer setup
 
 - `./git-init-submodules.sh` to preload all the relevent submodules after a git clone
+- copy core classes from submodule javacommons-core (in JavaCommons-core/src/main/java/picoded/core)
```
Comment (PicoCreator) [inline README.md]: not so sure if copy is the right word : maybe "load the submodule for javacommons-core"

Comment (rstai1130) [reply README.md]: updated the relevant note in https://github.com/picoded/JavaCommons-pdf/pull/1/commits/fd634209055d33c199473d2d6f11fe964f8e5785

File: build.gradle (hunk 1 above)
Comment (rstai1130) [reply build.gradle]: uncomment in https://github.com/picoded/JavaCommons-pdf/pull/1/commits/43f59fd2c2a51a86e482266c63cf264a1e2e7f0d

File: build.gradle (hunk 2 above)
Comment (rstai1130) [reply build.gradle:91]: switched to use mvnrepo batik libs ( https://github.com/picoded/JavaCommons-pdf/pull/1/commits/532e9ab83797a7a512f9c5f87f17e1b5b50182b2 ) and remove unused ( https://github.com/picoded/JavaCommons-pdf/pull/1/commits/d4954e2d10ee442c7f4a3bc3a430dd13400e966a )

**Pull request merged**